src/main/java/org/sqldroid/SQLDroidConnection.java

```diff
@@ -56,8 +58,8 @@
      * @param info Properties object with options.  Supported options are "timeout", "retry", and "shared".
      */
     public SQLDroidConnection(String url, Properties info) throws SQLException {
-        Log.v("SQLDroidConnection: " + Thread.currentThread().getId() + " \"" + Thread.currentThread().getName() + "\" " + this);
-        Log.v("New sqlite jdbc from url '" + url + "', " + "'" + info + "'");
+        Log.v("SQLDroid", "SQLDroidConnection: " + Thread.currentThread().getId() + " \"" + Thread.currentThread().getName() + "\" " + this);
```

Comment:
Yes, that's true but this changes are in Java code in order to make it compiles. We can fix it when move the code to Scala. Makes sense?